app/assets/stylesheets/administrate/application.scss

```diff
@@ -10,3 +10,5 @@
 @import "library/**/*";
 @import "base/**/*";
 @import "components/**/*";
+
+@import "administrate/customize";
```

Comment:
Yeah, I think the Sass lookup order is something along the lines of:

1. Current dir (i.e. `GEM_PATH/app/assets/stylesheets/administrate`)
2. Application asset path (i.e. `RAILS_ROOT/app/assets/stylesheets`)
3. Gem asset path (i.e. `GEM_PATH/app/assets/stylesheets`)

I tested all the variations manually and can confirm that the way it is now is correct, as it will check first the app assets and then the gem assets (2 & 3) in the proper order.